packages/material-ui-lab/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -101,6 +101,16 @@ export interface AutocompleteProps
    * @returns {ReactNode}
    */
   renderTags?: (value: any, state: RenderValueState) => React.ReactNode;
+  /**
+   * Object of titles to display when hovering the arrow or clear buttons.
+   * Options: openPopup, closePopup, clearPopup
+   * Default values: 'Open popup', 'Close popup', 'Clear'
```

Comment:
The type already shows the options so I don't see the point in writing it in the comment as well and the default values are extracted from the source (to show in the documentation) so it shouldn't be declared here.

Reply:
Oh I didn't know about the default value extraction. Fixed it.